components/report-details/support/dialog.tsx

```diff
@@ -140,25 +143,45 @@ const SupportReport = ({
 		setOpen,
 	};
 
-	if (isDesktop) {
-		return (
-			<ReportSupportUI
-				{...props}
-				UIComponent={Dialog}
-				ContentComponent={DialogContent}
-				HeaderComponent={DialogHeader}
-			/>
-		);
-	}
+	// if (isDesktop) {
```

Comment:
Delete commented out code if we aren't going to use it.

Did you check on this for for both desktop and mobile?

Reply:
Yes